### ISSUE TITLE
Doesn't work in new ember 3.24.0 app: bs-accordion.js: argument to macroCondition must be statically analyzable

### ISSUE THREAD

**Issue by drauschenbach** [kaliber5/ember-bootstrap#1442]
In a fresh new Ember 3.24.0 app, I perform `ember install ember-bootstrap`, and then `ember serve`. Startup fails with:

```sh
Proxying to http://__redacted__
Build Error (broccoli-persistent-filter:Babel > [Babel: ember-bootstrap]) in ember-bootstrap/components/bs-accordion.js

/Users/__redacted__/ember-bootstrap/components/bs-accordion.js: argument to macroCondition must be statically analyzable


Stack Trace and Error Report: /var/folders/fq/5bpv73rd6lb8qt9fcbd7p3vh0000gn/T/error.dump.e76cba2d9ce665b5835fbdc86fe08bb0.log
node:internal/process/promises:218
triggerUncaughtException(err, true /* fromPromise */);
^

BuildError: /Users/__redacted__/ember-bootstrap/components/bs-accordion.js: argument to macroCondition must be statically analyzable
at macroIfExpression (/Users/__redacted__/node_modules/ember-bootstrap/node_modules/@embroider/macros/src/glimmer/macro-condition.js:43:15)
at Object.macroIfMustache (/Users/__redacted__/node_modules/ember-bootstrap/node_modules/@embroider/macros/src/glimmer/macro-condition.js:54:18)
at MustacheStatement (/Users/__redacted__/node_modules/ember-bootstrap/node_modules/@embroider/macros/src/glimmer/ast-transform.js:155:50)
at visitNode (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7787:16)
at visitArray (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7878:20)
at visitKey (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7855:7)
at visitNode (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7809:9)
at visitKey (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7858:20)
at visitNode (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7809:9)
at visitArray (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7878:20) {
isBuilderError: true,
isSilent: undefined,
isCancellation: undefined,
broccoliPayload: {
originalError: Error: /Users/__redacted__/ember-bootstrap/components/bs-accordion.js: argument to macroCondition must be statically analyzable
at macroIfExpression (/Users/__redacted__/node_modules/ember-bootstrap/node_modules/@embroider/macros/src/glimmer/macro-condition.js:43:15)
at Object.macroIfMustache (/Users/__redacted__/node_modules/ember-bootstrap/node_modules/@embroider/macros/src/glimmer/macro-condition.js:54:18)
at MustacheStatement (/Users/__redacted__/node_modules/ember-bootstrap/node_modules/@embroider/macros/src/glimmer/ast-transform.js:155:50)
at visitNode (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7787:16)
at visitArray (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7878:20)
at visitKey (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7855:7)
at visitNode (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7809:9)
at visitKey (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7858:20)
at visitNode (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7809:9)
at visitArray (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7878:20) {
code: 'BABEL_TRANSFORM_ERROR',
file: 'ember-bootstrap/components/bs-accordion.js',
treeDir: '/var/folders/fq/5bpv73rd6lb8qt9fcbd7p3vh0000gn/T/broccoli-69452est86sQrW5N4/out-197-analyzer_ember_auto_import_analyzer'
},
originalMessage: '/Users/__redacted__/ember-bootstrap/components/bs-accordion.js: argument to macroCondition must be statically analyzable',
nodeId: 198,
nodeLabel: 'broccoli-persistent-filter:Babel > [Babel: ember-bootstrap] (Babel: ember-bootstrap)',
nodeName: 'broccoli-persistent-filter:Babel > [Babel: ember-bootstrap]',
nodeAnnotation: 'Babel: ember-bootstrap',
instantiationStack: '    at Babel.Plugin (/Users/__redacted__/node_modules/broccoli-plugin/index.js:7:31)\n' +
'    at Babel.Filter [as constructor] (/Users/__redacted__/node_modules/broccoli-babel-transpiler/node_modules/broccoli-persistent-filter/index.js:87:10)\n' +
'    at new Babel (/Users/__redacted__/node_modules/broccoli-babel-transpiler/index.js:58:10)\n' +
'    at Class.transpileTree (/Users/__redacted__/node_modules/ember-cli-babel/index.js:67:16)\n' +
'    at Object.toTree (/Users/__redacted__/node_modules/ember-cli-babel/index.js:77:30)\n' +
'    at /Users/__redacted__/node_modules/ember-cli-preprocess-registry/preprocessors.js:188:26\n' +
'    at Array.forEach (<anonymous>)\n' +
'    at processPlugins (/Users/__redacted__/node_modules/ember-cli-preprocess-registry/preprocessors.js:186:11)\n' +
'    at Function.module.exports.preprocessJs (/Users/__redacted__/node_modules/ember-cli-preprocess-registry/preprocessors.js:179:10)\n' +
'    at Class.preprocessJs (/Users/__redacted__/node_modules/ember-cli/lib/models/addon.js:1297:25)',
location: {
file: 'ember-bootstrap/components/bs-accordion.js',
treeDir: '/var/folders/fq/5bpv73rd6lb8qt9fcbd7p3vh0000gn/T/broccoli-69452est86sQrW5N4/out-197-analyzer_ember_auto_import_analyzer',
line: undefined,
column: undefined
},
error: {
message: '/Users/__redacted__/ember-bootstrap/components/bs-accordion.js: argument to macroCondition must be statically analyzable',
stack: 'Error: /Users/__redacted__/ember-bootstrap/components/bs-accordion.js: argument to macroCondition must be statically analyzable\n' +
'    at macroIfExpression (/Users/__redacted__/node_modules/ember-bootstrap/node_modules/@embroider/macros/src/glimmer/macro-condition.js:43:15)\n' +
'    at Object.macroIfMustache (/Users/__redacted__/node_modules/ember-bootstrap/node_modules/@embroider/macros/src/glimmer/macro-condition.js:54:18)\n' +
'    at MustacheStatement (/Users/__redacted__/node_modules/ember-bootstrap/node_modules/@embroider/macros/src/glimmer/ast-transform.js:155:50)\n' +
'    at visitNode (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7787:16)\n' +
'    at visitArray (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7878:20)\n' +
'    at visitKey (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7855:7)\n' +
'    at visitNode (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7809:9)\n' +
'    at visitKey (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7858:20)\n' +
'    at visitNode (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7809:9)\n' +
'    at visitArray (/Users/__redacted__/node_modules/ember-source/dist/ember-template-compiler.js:7878:20)',
errorType: 'Build Error',
codeFrame: '/Users/__redacted__/ember-bootstrap/components/bs-accordion.js: argument to macroCondition must be statically analyzable',
location: {
file: 'ember-bootstrap/components/bs-accordion.js',
treeDir: '/var/folders/fq/5bpv73rd6lb8qt9fcbd7p3vh0000gn/T/broccoli-69452est86sQrW5N4/out-197-analyzer_ember_auto_import_analyzer',
line: undefined,
column: undefined
}
},
broccoliNode: {
nodeName: 'broccoli-persistent-filter:Babel > [Babel: ember-bootstrap]',
nodeAnnotation: 'Babel: ember-bootstrap',
instantiationStack: '    at Babel.Plugin (/Users/__redacted__/node_modules/broccoli-plugin/index.js:7:31)\n' +
'    at Babel.Filter [as constructor] (/Users/__redacted__/node_modules/broccoli-babel-transpiler/node_modules/broccoli-persistent-filter/index.js:87:10)\n' +
'    at new Babel (/Users/__redacted__/node_modules/broccoli-babel-transpiler/index.js:58:10)\n' +
'    at Class.transpileTree (/Users/__redacted__/node_modules/ember-cli-babel/index.js:67:16)\n' +
'    at Object.toTree (/Users/__redacted__/node_modules/ember-cli-babel/index.js:77:30)\n' +
'    at /Users/__redacted__/node_modules/ember-cli-preprocess-registry/preprocessors.js:188:26\n' +
'    at Array.forEach (<anonymous>)\n' +
'    at processPlugins (/Users/__redacted__/node_modules/ember-cli-preprocess-registry/preprocessors.js:186:11)\n' +
'    at Function.module.exports.preprocessJs (/Users/__redacted__/node_modules/ember-cli-preprocess-registry/preprocessors.js:179:10)\n' +
'    at Class.preprocessJs (/Users/__redacted__/node_modules/ember-cli/lib/models/addon.js:1297:25)'
},
versions: { 'broccoli-builder': '3.5.1', node: 'v15.1.0' }
}
}
```

---

Output from `ember bootstrap:info`:

```
Npm packages:
ember-bootstrap: ^4.6.3 -> 4.6.3
ember-cli: ~3.24.0 -> 3.24.0
bootstrap: ^4.6.0 -> 4.6.0
bootstrap-sass: n/a
ember-cli-sass: ^10.0.1 -> 10.0.1
ember-cli-less: n/a

Bower packages:
bootstrap: n/a
bootstrap-sass: n/a

ember-bootstrap configuration:
bootstrapVersion: 4
importBootstrapCSS: false
```

**Comment by jelhan** [comment]
This is a big in ember-cli-htmlbars v5.6.0. See https://github.com/kaliber5/ember-bootstrap/pull/1441#issuecomment-786861809 It should be fixed in ^5.6.2

**Comment by simonihmig** [comment]
Closing this, assuming this problem is indeed fixed now upstream. Please reopen if still experiencing problems!
